core/templates/pages/contributor-dashboard-page/translation-badge/translation-badge.component.html

```diff
@@ -0,0 +1,55 @@
+<div *ngIf="isUnlocked" class="badge-container">
```

Comment:
I would expect that we have some kind of base badge object and then have a question and translation badge to inherit from that. Why isn't that so?

Reply:
Created a new component